main.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+
+moist_item = Item()
+trenisha = Vendor()
```

Comment:
😆  hahaha I'm glad you were able to follow along with the example. Always funny to see 'moist' show up 😆